src/main/webapp/app/exercises/programming/shared/utils/programming-exercise.utils.ts

```diff
@@ -13,6 +13,9 @@ import { AssessmentType } from 'app/entities/assessment-type.model';
 
 const BAMBOO_RESULT_LEGACY_TIMESTAMP = 1557526348000;
 
+/*
+ * This is a legacy check, results before the 24th May are considered legacy.
```

Comment:
Of which year?

Reply:
Its in a comment in the beginning of this class